### PR TITLE
코드 설명용 (merge x)

### PR DESCRIPTION
**Pull request by hagiss**
pretrain : python3 pl_train.py --name vit
--name은 tensor board logger가 checkpoint와 여러 값들을 저장하는 path입니다
linear : python3 fine_tune.py -ch [checkpoint path] --name [name]

### CODE REVIEW

File: pl_train.py [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+        loss2 = self.criterion(logits2, labels)
+        loss = loss1 + loss2
+        loss /= 2
+        return loss
+
+    def training_step(self, batch, _):
+        aug1 = self.augment1(batch[0])
+        aug2 = self.augment2(batch[0])
+        output_loss = self.forward(torch.cat((aug1, aug2), dim=0))
+        # layer_loss = self.get_layer_loss() / (self.depth - 1)
```
Comment (hagiss) [inline pl_train.py:104]: 여기 주석제거하면 layer loss 추가가능합니다
